app/templates/eventView.html

```diff
@@ -1,40 +1,23 @@
 {% set title = eventData.name %}
-{% extends "base.html" %}
+
+{% set eventName = eventData.name %}
+{% set eventProgram = eventData.program %}
+{% set eventProgramName = eventData["program"].programName %}
+{% set eventPast = event.isPast %}
+{% set eventID = eventData.id %}
+
+{% set tabName = 'view' %}
+
```

Comment:
I'd remove the blank line between these statements to show that `tabName` is connected to the extends